CODE_REVIEW_CHECKLIST.md

```diff
@@ -69,3 +71,10 @@ may have.
 
 - [ ] Will this change work on all of our supported platforms and
       browsers?
+
+- [ ] Do we expect significant changes in performance due to the
```

Comment:
This is very "scrutinizey": I think both this point and the one above are important-- is there a place that documents how to go about verifying supported platforms/performance impact? Should it be expected that if a code-writer has seen these that s/he/it note that on the PR in anticipation of the question?